src/hmem_cuda.c

```diff
@@ -301,6 +445,32 @@ int cuda_hmem_init(void)
 {
 	int ret;
 	int gdrcopy_ret;
+	int cuda_enable_xfer;
```

Comment:
This is used as a parameter to `fi_param_get_bool`, which takes an `int *` as the argument.

Reply:
Sorry yes that part is fine. I meant to save it for later. This one looks like it's local so it's fine. The other one below is global so that's why I think it makes more sense as a bool but it's not a big deal. The other providers do something like this:
```
        fi_param_get_bool(&rxm_prov, "enable_dyn_rbuf", &ret);
        domain->dyn_rbuf = (ret != 0);
```